diff --git a/common/compiled/node/auth/keyv.js b/common/compiled/node/auth/keyv.js
index 2bf486f..64dd4d1 100644
--- a/common/compiled/node/auth/keyv.js
+++ b/common/compiled/node/auth/keyv.js
@@ -1,8 +1,8 @@
 let keyv;
 const setTokenService = service => (keyv = service);
-const setRefreshToken = async (id, refresh_token) => keyv.set(id, refresh_token);
-const getRefreshToken = async id => keyv.get(id);
-const revokeRefreshToken = async id => keyv.delete(id);
+const setRefreshToken = async (id, refresh_token) => keyv.set(String(id), refresh_token);
+const getRefreshToken = async id => keyv.get(String(id));
+const revokeRefreshToken = async id => keyv.delete(String(id));
 
 const setUserService = () => {};
 const setRefreshTokenStoreName = () => {};
